fun-token-contract/.cargo/config

```diff
@@ -0,0 +1,2 @@
+[build]
+rustflags = ["-C", "link-args=-s"]
```

Comment:
let's put the whole thing in `/devx` directory. This contract is not part of the principal project, it's used for testing and simulation.

Reply:
It's really complex as it is. Let's leave it here until the core CLP functions pass all the integration tests